django_codemod/visitors/crypto.py

```diff
@@ -0,0 +1,22 @@
+from typing import Sequence
+
+from libcst import Arg, Call, Integer
+
+from django_codemod.constants import DJANGO_3_0, DJANGO_4_0
+from django_codemod.visitors.base import BaseFuncRenameTransformer
+
+
+class GetRandomStringTransformer(BaseFuncRenameTransformer):
+    """Update lengthless uses of get_random_string()"""
+
+    deprecated_in = DJANGO_3_0
+    removed_in = DJANGO_4_0
+    rename_from = "django.utils.crypto.get_random_string"
+    rename_to = "django.utils.crypto.get_random_string"
+
+    def update_call_args(self, node: Call) -> Sequence[Arg]:
+        # TODO: support adding length when keyspace is present
```

Comment:
I guess we could check if the first argument is a keyword argument with the keyword `allowed_chars`, and in this case, we need to update the list of arguments.

I'm doing the opposite for the deprected `weak` argument for signals `disconnect()`, but you could use a similar check:

https://github.com/browniebroke/django-codemod/blob/2e81ab4ec3407ea37ba6e03aa610743be5d22c01/django_codemod/visitors/signals.py#L84

Let me know if you want to address it in this pull request. If you don't have much more time, I don't mind fixing it later.

Reply:
Added that in, with a snazzy new `find_keyword_arg` utility function to boot 😄